### PR TITLE
OCD-1830 - take out QA reports

### PR DESCRIPTION
**Pull request by brianlindsy**

### CODE REVIEW

File: src/app/admin/components/reports/reports.directive.js [hunk 1]
```diff
@@ -405,21 +392,15 @@
...
                         var statusActivity = angular.copy(activity);
                         statusActivity.details = statusChange;
                         output.status.push(statusActivity);
-
-                        // count status change as questionable
-                        activity.details.push('<span class="bg-danger"><strong>' + statusChange + '</strong></span>');
-                        activity.questionable = true;
                     }
                     if (data[i].newData.certificationEdition.name === '2011') {
                         activity.action = '<span class="bg-danger">' + activity.action + '</span>';
```
Comment (andlar) [inline src/app/admin/components/reports/reports.directive.js]: Giving things the class `bg-danger` is how things are depicted as being questionable. This line indicates an activity is questionable on the screen, though we're not parsing that any more

File: src/app/admin/components/reports/reports.directive.js [hunk 2]
```diff
@@ -624,30 +598,23 @@
...
         }
 
-        function compareCerts (prev, curr, questionable) {
+        function compareCerts (prev, curr) {
             var ret = [];
             var change;
             var certKeys = [
                 {key: 'apiDocumentation', display: 'API Documentation'},
-                {key: 'g1Success', display: 'Certified to G1', questionable: true},
-                {key: 'g2Success', display: 'Certified to G2', questionable: true},
```
Comment (andlar) [inline src/app/admin/components/reports/reports.directive.js]: Are you not checking g1/g2 Success flags any more?

File: src/app/admin/components/reports/reports.directive.spec.js [hunk 3]
```diff
@@ -532,24 +504,6 @@
                         expect(vm.searchedCertifiedProducts[0]).toEqual(expectedActivity);
                     });
 
-                    it('should handle criteria addition', function () {
```
Comment (andlar) [inline src/app/admin/components/reports/reports.directive.spec.js]: These two tests should be removed, as the system still needs to get the right output for criteria addition/removal, even if we're not treating them as questionable any more

File: src/app/admin/components/reports/reports.directive.js (hunk 1 above)
Comment (brianlindsy) [reply src/app/admin/components/reports/reports.directive.js]: So that questionable flag on the activity object should still be in there?

Comment (andlar) [reply src/app/admin/components/reports/reports.directive.js]: No, we're getting rid of the questionable flag, but I'm looking at line 421, where the action is highlighted using the class "bg-danger"

File: src/app/admin/components/reports/reports.directive.js [hunk 4]
```diff
@@ -133,6 +133,13 @@
                 case 'cp-surveillance':
                 case 'cp-cap':
                 case 'cp-other':
+                case 'cp-questionable':
```
Comment (andlar) [inline src/app/admin/components/reports/reports.directive.js]: We don't have this case any more. There is no navigation to the "questionable activities" options

File: src/app/admin/components/reports/reports.directive.js [hunk 5]
```diff
@@ -398,6 +405,8 @@
                         var statusActivity = angular.copy(activity);
                         statusActivity.details = statusChange;
                         output.status.push(statusActivity);
+
+                        activity.details.push('<span class="bg-danger"><strong>' + statusChange + '</strong></span>');
```
Comment (andlar) [inline src/app/admin/components/reports/reports.directive.js]: Why are you marking this activity as questionable? That's what the class "bg-danger" implies...

File: src/app/admin/components/reports/reports.directive.js [hunk 6]
```diff
@@ -398,6 +405,8 @@
...
                     }
                     if (data[i].newData.certificationEdition.name === '2011') {
                         activity.action = '<span class="bg-danger">' + activity.action + '</span>';
```
Comment (andlar) [inline src/app/admin/components/reports/reports.directive.js]: Why is this marked as "bg-danger"? That means "questionable"

File: src/app/admin/components/reports/reports.directive.spec.js [hunk 7]
```diff
@@ -504,6 +504,24 @@
                         expect(vm.searchedCertifiedProducts[0]).toEqual(expectedActivity);
                     });
 
+                    it('should handle criteria addition', function () {
```
Comment (andlar) [inline src/app/admin/components/reports/reports.directive.spec.js]: Did you run these tests? They shouldn't pass, as the html that's being output has the "bg-danger" class, which should not happen any more, as we're not marking things as questionable in the UI

File: src/app/admin/components/reports/reports.directive.js (hunk 6 above)
Comment (brianlindsy) [reply src/app/admin/components/reports/reports.directive.js]: Ah ok, I was confused, I thought you meant in your previous comment that this line should be added back in

Comment (andlar) [reply src/app/admin/components/reports/reports.directive.js]: There were two basic things that needed to be removed for this ticket; the first was the actual "questionable activities" report, which would have entailed some navigation changes, a chunk of HTML deleted, and removing wherever the code was checking for questionable stuff to fill the list of activities.

The second part was un-highlighting questionable stuff wherever it showed up. Anything that was questionable before was highlighted in the various "other" Certified Product reports, and likely in the Version/Product reports too, and that highlight needed to be removed wherever it was found

File: src/app/admin/components/reports/reports.directive.js [hunk 8]
```diff
@@ -406,10 +405,6 @@
                         statusActivity.details = statusChange;
                         output.status.push(statusActivity);
 
-                        activity.details.push('<span class="bg-danger"><strong>' + statusChange + '</strong></span>');
```
Comment (andlar) [inline src/app/admin/components/reports/reports.directive.js]: Why aren't you pushing the statusChange to the regular activity field? It's currently there, so it should stay there after removing the "questionable" flag / display tweak

**Pull request merged**